dataminer-overview/DataMiner_Hosting_and_HA/DM_and_storage_selfhosted.md

```diff
@@ -0,0 +1,40 @@
+
+In terms of **redundancy**, each DMA (DataMiner node) in a DataMiner System can be teamed up with a dedicated backup DMA. Within a DMA redundancy team, the backup DMA will continuously be synchronized so that, at all times, it is ready to take over from its team member the moment that one fails. When you team up a particular DataMiner node with a backup DMA, a virtual DMA team will be created. Within a DMA team, the two team members will act as peers. In other words, they will not act as master DMA versus slave DMA, but as active DMA versus passive DMA.
+
+By default, all synchronization traffic between the active and the passive team member will pass via the corporate network. If necessary, both team members can also be equipped with an additional network card to be used for synchronization traffic only.
+
+The decision when to switch from the active to the passive DMA can be taken either by a person (i.e. manual Failover) or by the DMAs themselves (i.e. automatic Failover). In the latter case, the two DMAs in the team will check each other’s status by exchanging heartbeats.
+
+> [!TIP]
+> See also: [Failover User Guide](xref:failover)
+
+In terms of **backup and restore**, DataMiner nodes require a one-time configuration by the administrator in order to fully automatically take care of backups. The administrator can choose:
+
+- When the DMAs need to take a backup (e.g. every 24 hours at 01:00 AM).
+- What kind of information needs to be included in those backups (configuration, core software, connectors, historical alarms, etc.).
+- How many backups to maintain (e.g. 7 backups, with a backup taken every 24 hours, so that you can always go back one week).
+- Where to store the backup file. The backup file can be stored locally on the DMA (e.g. on a second hard drive), but it can also be stored on a secured network drive on a file server.
+
+A default DataMiner System also comes with a set of features and capabilities that increase the availability and that enable proactive maintenance and support. This includes for example:
+
+- Automatic and user-definable reboot procedure.
+- Generic watchdog strategy.
+- Automatic collection and email forwarding of fault and logging information.
+- Automated backup.
+- Scheduled email health reporting.
+- Self-maintaining database with user-definable settings.
+- Etc.
+
+From a storage point of view, DataMiner is equipped with Cassandra and Elasticsearch storage solutions. These databases can be run on premises, in a private cloud, hybrid or in a public cloud. Cassandra and Elasticsearch clustering capabilities are supported.
```

Comment:
What about OpenSearch? This is also supported, and to my knowledge it may even become the default some time in the future.